src/main/scala/org/scalajs/cli/Scalajsld.scala

```diff
@@ -172,13 +169,15 @@ object Scalajsld {
 
       val linker = StandardLinker(config)
       val logger = new ScalaConsoleLogger(options.logLevel)
-      val outFile = new WritableFileVirtualBinaryFile(options.output)
+      val outFile = new WritableFileVirtualBinaryFile(options.output.toPath())
       val output = LinkerOutput(outFile)
       val cache = (new IRFileCache).newCache
 
-      val future = linker.link(cache.cached(irContainers), moduleInitializers,
-          output, logger)
-      Await.result(future, Duration.Inf)
+      val result = FileScalaJSIRContainer
+        .fromClasspath(classpath)
+        .flatMap(cache.cached(_))
```

Comment:
Consider (in another commit?) to replace this with `.flatMap(_.sjsirFiles)`. Caching really doesn't make any sense here...

Reply:
Good point. I amended the commit, because it is actually related to the upgrade: before, there was no easy way to go from containers to IR files, other than using an IR cache. Now it's much easier.